bin/opam.ml

```diff
@@ -208,30 +220,16 @@ let submit ~token ~dry_run ~yes ~opam_repo ~user local_repo remote_repo pkgs
     | Some { owner; repo } -> Text.rewrite_github_refs ~user:owner ~repo changes
     | None -> changes
   in
-  let user =
-    match user with
-    | Some user -> Ok user (* from the .yaml configuration file *)
-    | None -> (
-        match Github.Parse.user_from_remote remote_repo with
```

Comment:
All uses of this functions have been removed, we can remove it from github.ml

Reply:
Good catch!